CHANGELOG.md

```diff
@@ -0,0 +1,76 @@
+## Unreleased
+
+No changes yet :)
+
+## v6.0.0-beta.3
+
+### Potentially breaking changes
+
+- #547 requires your integration to understand the network `ByzantiumToConstantinopleAt5`, which is analogous to existing pseudo networks such as `FrontierToHomesteadAt5` and `EIP158ToByzantiumAt5`. It also changed the format of some blockchain tests, [keys of "0x" are now "0x00"](https://github.com/ethereum/tests/pull/547/files#diff-d31f2819c1febd4c8d673a15a8375af5)
```

Comment:
0x to 0x00 was done to the filler. It doesn't affect anything. Its just that 0x  confuses me. No need to mention this change as it might confuse. Because real 0x requirement done to RLP tests

Reply:
Good point, I'll remove it.

Reply:
Fixed